controllers/profilecatalogsource_controller.go

```diff
@@ -63,6 +77,26 @@ func (r *ProfileCatalogSourceReconciler) Reconcile(ctx context.Context, req ctrl
 	logger.Info("updating catalog entries", "profiles", pCatalog.Spec.Profiles)
 	r.Profiles.Update(pCatalog.Name, pCatalog.Spec.Profiles...)
 
+	gitRepoManager := gitrepository.NewManager(ctx, pCatalog.Namespace, r.Client, time.Minute*2, time.Second*5)
```

Comment:
what are the last 2 args? can we have consts or smth for clarity?

Reply:
fixed